src/Passes/libs/QubitAllocationAnalysis/SPECIFICATION.md

```diff
@@ -0,0 +1,9 @@
+# Qubit Allocation Analysis
```

Comment:
This specification.md is filled in, but the other two are not. Maybe include some placeholder text in the other two files indicating that they will be written in the future?

Reply:
That is a good suggestion. I will add a placeholder text to the template in one of the coming PRs.